logging/logging.go

```diff
@@ -0,0 +1,82 @@
+package logging
+
+import (
+	"fmt"
+	"log"
+)
+
+type LogLevel int
+
+func (level LogLevel) String() string {
+	return []string{"ERROR", "WARN", "INFO", "DEBUG", "TRACE"}[level]
+}
+
+const (
+	ErrorLevel LogLevel = iota
+	WarningLevel
+	InfoLevel
+	DebugLevel
+	TraceLevel
+)
+
+func init() {
+	log.SetFlags(0) // Removes all logger prefixes
+}
+
+func formatLogLevel(logLevel LogLevel) string {
+	return fmt.Sprintf("[%s]", logLevel)
+}
+
+func prefixLogArgs(logLevel string, args ...interface{}) []interface{} {
+	return append([]interface{}{logLevel}, args...)
+}
+
+func Error(args ...interface{}) {
+	log.Println(prefixLogArgs(formatLogLevel(ErrorLevel), args...)...)
```

Comment:
I'm not 100% sure that this works in go, but wouldn't it be easier to have something like:

```suggestion
	log.Println(formatLogLevel(ErrorLevel), args...)
```